app/src/main/res/layout/solution_summary.xml

```diff
@@ -130,32 +132,15 @@
           android:paddingBottom="16dp"
           android:visibility="@{isListExpanded? View.VISIBLE : View.GONE}">
 
-          <LinearLayout
-            android:layout_width="match_parent"
+          <TextView
```

Comment:
I think instead of :-`app:cardBackgroundColor=component_color_hints_and_solution_summary_card_background_color` use `app:cardBackgroundColor=component_color_shared_screen_primary_background_color`


`component_color_hints_and_solutions_fragment_text_color` to `component_color_shared_primary_text_color`


`component_color_hints_and_solution_summary_button_text_color`   to  `component_color_shared_secondary_4_text_color`


`component_color_hints_and_solution_summary_divider_color`  to  `component_color_shared_divider_color`


`component_color_hints_and_solutions_fragment_description_color`  to  `component_color_shared_primary_text_color`

Reply:
Actually, the main thinking behind moving this to shared colors is that while me and @rt4914 working on the dark-mode so we noticed there are so many commonly used colors. And to reduce Duplicacy and make the color file more short and easy to find the exact color.